test/AutoRest.TestServerLowLevel.Tests/dpg_initial.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using AutoRest.TestServer.Tests.Infrastructure;
+using dpg_initial_LowLevel;
+using NUnit.Framework;
+
+namespace AutoRest.TestServer.Tests
+{
+    public class DpgInitialTest : TestServerLowLevelTestBase
+    {
+        [Test]
+        public Task DpgAddOptionalInput_NoParams_Test() => Test(async (host) =>
+        {
+            var result = await new ParamsClient(Key, host).HeadNoParamsAsync();
+            Assert.IsEmpty(result.Content.ToArray());
+            Assert.AreEqual(200, result.Status);
+            Assert.AreEqual(123, result.Headers.ContentLength);
+            var result2 = await new ParamsClient(Key, host).HeadNoParamsAsync(new Azure.RequestContext());
```

Comment:
nit: would it be possible to separate test cases with carriage returns?  It would make them reasier to read.